src/vaishnavi_SeleniumBasics/GoIbibo.java

```diff
@@ -0,0 +1,136 @@
+		wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//div[contains(@class,'dF whiteBg')]/ul//li")));
+		int count = driver.findElements(By.xpath("//div[contains(@class,'dF whiteBg')]/ul//li")).size();
+		
+		for(int index=1;index<=count;index++) {
+			String suggestion = driver.findElement(By.xpath("//div[contains(@class,'dF whiteBg')]/ul//li["+index+"]/div/div[@class='dib marginL10 pad0 textOverflow width90']/div[1]/span[1]")).getText();
+			autoSuggestions.add(suggestion);						
+		}
+		return autoSuggestions;
+	}
+	
+	// Method to print auto suggestions after we type Pun in from field and Click on Pune
+	@Test(priority =1 )
+	void getFromAutoSuggestions() {
+		driver.findElement(By.xpath("//input[@id='gosuggest_inputSrc']")).sendKeys("Pun"); //Enter Pun in Form field
+		ArrayList<String> fromSuggestions = verifyAutoSuggestions(); //Call another method to get the list
+		System.out.println("From autoSuggestions are as below: ");
+		System.out.println(fromSuggestions);
+		for(int index=0;index<fromSuggestions.size();index++) {
+			if(fromSuggestions.get(index).equals("Pune, India")) {
+				WebElement expected = driver.findElement(By.xpath("//div[contains(@class,'dF whiteBg')]/ul//li["+(index+1)+"]/div/div[@class='dib marginL10 pad0 textOverflow width90']/div[1]/span[1]"));
+				expected.click();
+			}			
+		}
+	}
+	
+	// Method to print auto suggestions after we type Del in Destination field and click on Delhi
+	@Test(priority=2)
+	void getDestinationAutoSuggestions() {
+		driver.findElement(By.xpath("//input[@id='gosuggest_inputDest']")).sendKeys("Del"); //Enter Del in Destination field
+		ArrayList<String> destinationSuggestions = verifyAutoSuggestions(); //Call another method to get the list
+		System.out.println("Destination autoSuggestions are as below: ");
+		System.out.println(destinationSuggestions);
+		for(int index=0;index<destinationSuggestions.size();index++) {
+			if(destinationSuggestions.get(index).equals("Delhi, India")) {
+				WebElement expected = driver.findElement(By.xpath("//div[contains(@class,'dF whiteBg')]/ul//li["+(index+1)+"]/div/div[@class='dib marginL10 pad0 textOverflow width90']/div[1]/span[1]"));
+				expected.click();
+			}			
+		}
+	}
+	
+	// Method for Departure calendar Validation
+	@Test(priority=3)
+	void calendarDepartureValidation() {
+		String expected = "September 2020";
+		String actual = driver.findElement(By.xpath("//div[@role='heading']")).getText();
+		assertEquals(expected, actual); //Verify month-year text is correct
+	   
+		int count = driver.findElements(By.xpath("//div[@class='DayPicker-Weekday']")).size();
+		for(int index=1;index<=count;index++) {
+		    WebElement days = driver.findElement(By.xpath("//div[@class='DayPicker-Weekday']["+index+"]/abbr"));
+		    days.isDisplayed();  //  Mon to Sun - days are visible
+		}
+		
+		WebElement date = driver.findElement(By.xpath("//div[@id='fare_20200911']"));
+		date.isEnabled(); //Select date should be enabled
```

Comment:
Here you should write in if block or this validation is not valid. 
Also expected date validation using system date.